src/app/owner/page.tsx

```diff
@@ -0,0 +1,76 @@
+import Image, { StaticImageData } from 'next/image';
+import about from "@images/about/about1.png";
+import AboutHeading from './AboutHeading'; 
+import SubHeading from './SubHeading';
+import Section from './Section';
+import workMethodsContent from '../utils/workMethodsContent';
+import { educationAndExperience, certificatesAndAchievements } from '../utils/aboutContent';
+
+const About = () => {
+  return (
+    <div className="min-h-screen bg-white">
+      <main className="max-w-4xl mx-auto p-4">
+        <nav className="text-[16px] mb-4 mt-[70px]">
+          <a href="/" className="font-semibold leading-6 underline" aria-label="Wróć do strony głównej">powrót</a>
+        </nav>
+
+        <p className="font-bold text-sm leading-6 mt-[29px]">O nas // Katarzyna Bober</p>
+        <h1 className="text-center font-dm-sans font-medium text-[44px] leading-[52.8px] tracking-[-0.04em] mt-[40px]">Katarzyna Bober</h1>
+
+        <div className="flex justify-center mb-[20px]">
+          <Image
+            src={about as StaticImageData}
+            alt="Katarzyna Bober with dog"
+            width={1217}
+            height={600}
+            className="rounded-lg mt-[100px] max-w-full"
+            priority
+          />
+        </div>
+
+        <div className="font-roboto font-normal text-sm leading-6 text-left mb-[80px] mt-[100px]">
```

Comment:
"font-roboto" does not apply in the code, it also seems to me that it has been withdrawn from the project.